### DISCUSSION TITLE
ELRS 2.0 Signal drop

### DISCUSSION THREAD

**Discussion by PapaDrone** [ExpressLRS/ExpressLRS#1192] [Q&A]
December 16, 2021
I updated the BetaFPV 900mhz module to ELRS 2.0 but signal drops at about 20 yards. Tried it on several different drones bujt same results. flashed back to ELRS 1.2.1 and no problems.

**Comment by edernucci** [comment]
@PapaDrone The same thing happened to me today. Please try the procedure described here https://www.expresslrs.org/2.0/quick-start/pre-1stflight/#bench-test and let me know if it passed. It will fatally be close to -70dB as it happened to me. If that's the case, we've had the same problem with the same equipment.

**Comment by SunjunKim** [comment]
I believe issue #1186 is a similar thing.

**Comment by SunjunKim** [comment]
Would you try cleanup caches and reflash, and try the test again?

![](https://media.discordapp.net/attachments/798006228450017290/916375460420669500/clearIODependencies.jpg)

**Comment by PapaDrone** [comment]
Thank you. I will try the suggestions this weekend. I will let you know if
they work.

**Comment by JBKingdon** [comment]
Many thanks to @edernucci for finding the problem and submitting PR https://github.com/ExpressLRS/ExpressLRS/pull/1198

The PR has been merged into the 2.0.x-maintenance branch, so if others who hit the problem are able to test with that it would be great to confirm that it resolves this issue for everyone.

**Reply by GHpaul23** [reply]
Having problem finding threads on ELRS and I have a weird situation.
I have a Mobula 7 1s with elrs, Betafpv 500 elrs module, in my Jumper t8sg v2.
After 5 hours on set up it was flying great for a week, 50 batteries.
Now it flys for 30 seconds then falls out of the air with blinking light on Mobula with no connection. If I unplug and plug in again, it is bound for 40 seconds just resting, then blinking light - no connection.
No comments anywhere discuss working elrs connections that drop out at close range after 30 -40 seconds.
fc is running 8k/2k.

Is there a thermal sensor that can shut the connection off? After losing connection, I need to wait a minute or two for FC to "cool off" before plugging in to reconnect (usb or battery). When connection is lost, if I unplug and plug in immediately it will still be unconnected.

I reloaded the Betafpv module, the Mobula came with v2.0 elrs and I have not done anything to that.
After 6 more hours of reading/watching videos, nobody has mentioned close range drop outs after 30 seconds.
